### ISSUE TITLE
Error while running DELETE query

### ISSUE THREAD

**Issue by mporracindie** [opendistro-for-elasticsearch/sql#58]
**Query im running:**
```
POST /_opendistro/_sql
{
  "query": "DELETE FROM competencia-response WHERE date_format(timestamp,'Y-MM-dd') <= '2019-05-10'"
}

```
**Error:**

```
{
  "took": 736,
  "timed_out": false,
  "total": 22243159,
  "updated": 0,
  "created": 0,
  "deleted": 0,
  "batches": 1,
  "version_conflicts": 1000,
  "noops": 0,
  "retries": {
    "bulk": 0,
    "search": 0
  },
  "throttled_millis": 0,
  "requests_per_second": -1,
  "throttled_until_millis": 0,
  "failures": [
    {
      "index": "competencia-response",
      "type": "doc",
      "id": "780e5671-bcd9-4908-9b8f-5a8dc74dd6a1",
      "cause": {
        "type": "version_conflict_engine_exception",
        "reason": "[doc][780e5671-bcd9-4908-9b8f-5a8dc74dd6a1]: version conflict, required seqNo [6579957], primary term [2]. current document has seqNo [49387751] and primary term [2]",
        "index_uuid": "ZwwuvGeLQXSyG8OJM_lv8g",
        "shard": "3",
        "index": "competencia-response"
      },
      "status": 409
    }
.
.
.
 ]
}
```

**Comment by abbashus** [comment]
Could you supply the data that you are trying to delete? This seems to be [Elasticsearch issue](https://discuss.elastic.co/t/version-conflict-while-using-delete-by-query/104831), but would like to reproduce.

**Comment by mporracindie** [comment]
@abbashus Maybe has to do with the fact that i upgraded opendistro? Sadly i can not share the data since its sencible.

**Comment by abbashus** [comment]
Thanks @mporracindie , will try to reproduce on our end.

**Comment by mporracindie** [comment]
Solved on #72
